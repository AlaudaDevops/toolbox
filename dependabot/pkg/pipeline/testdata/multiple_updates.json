[
    {
        "package_name": "golang.org/x/crypto",
        "current_version": "v0.0.1",
        "fixed_version": "v0.1.0",
        "vulnerability_ids": [
            "CVE-2023-1234"
        ],
        "language": "go",
        "success": true,
        "error": ""
    },
    {
        "package_name": "github.com/gorilla/websocket",
        "current_version": "v1.4.0",
        "fixed_version": "v1.5.0",
        "vulnerability_ids": [
            "CVE-2023-5678"
        ],
        "language": "go",
        "success": true,
        "error": ""
    }
]
