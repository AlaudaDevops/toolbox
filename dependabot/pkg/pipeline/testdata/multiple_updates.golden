fix: update 2 vulnerable dependencies

Security updates for multiple packages
